scripts/mkcloud

```diff
@@ -955,16 +955,18 @@ function onhost_reset_admin
 function crowbarupgrade_5plus
 {
     if iscloudver 6plus ; then
-        cloudsource=$upgrade_cloudsource onadmin prepare_cloudupgrade_nodes_repos_6_to_7
-        onadmin upgrade_ses_to_4
+        cloudsource=$upgrade_cloudsource onadmin prepare_cloudupgrade_nodes_repos
+        if [[ $cloudsource = develcloud6 ]]; then
```

Comment:
Should this be `if iscloudver 6 ; then` or would there be a reason to not upgrade SES for `cloudsource=GM6+up` ?
Could also be written as
```bash
iscloudver 6 && onadmin upgrade_ses_to_4
```

Reply:
I like `iscloudver 6 && onadmin upgrade_ses_to_4`